portality/templates/formcontext/admin_article_form.html

```diff
@@ -0,0 +1,62 @@
+<fieldset>
+    {{ render_field_horizontal(form.title, style="width: 400px", placeholder="Enter the article title here") }}
+    {{ render_field_horizontal(form.doi, style="width: 300px", placeholder="10.1234/article-32") }}
```

Comment:
@richard-jones do you have an idea what to do about it? Would you like to introduce the separate error (ex: "DOI or Fulltext URL is required.") when none of these fields is provided (I'm not sure but it might be not trivial) or simply change the error message? It's already quite long: `Invalid DOI. A DOI can optionally start with a prefix (such as "doi:"), followed by "10." and the remainder of the identifier` so I don't know how to construct it.

Reply:
Plus: the error might be slightly confusing when it says "Invalid..." in two places but we already have a description "(You must provide a DOI and/or a Full-Text URL)" so I am not sure if it's worth it to add another Error that will duplicate that. It's exactly the same as in Publisher Metadata form and users are not unhappy about it...

Reply:
Issue

Reply:
This is consistent with current publisher metadata form